Source/com/drew/lang/SequentialReader.java

```diff
@@ -386,4 +388,20 @@ public byte[] getNullTerminatedBytes(int maxLengthBytes) throws IOException
             System.arraycopy(buffer, 0, bytes, 0, length);
         return bytes;
     }
+
+
+    /**
+     * Read until the null terminated byte and automatically move the end of the requested position.
+     * @param maxLengthBytes
+     * @param charset
+     * @return
+     * @throws IOException
+     */
+    public StringValue getNullTerminatedStringAndSkipToNextPosition(int maxLengthBytes, Charset charset) throws IOException {
+        byte[] bytes = this.getNullTerminatedBytes(maxLengthBytes);
+        if (bytes.length < maxLengthBytes - 1) {
+            this.trySkip(maxLengthBytes - bytes.length - 1);
+        }
+        return new StringValue(bytes, charset);
+    }
```

Comment:
We don't need to add a new method. We can call `getNullTerminatedStringValue` instead.